crates/commands/src/commands/install.rs

```diff
@@ -85,6 +86,11 @@ pub struct Install {
     /// location.
     #[arg(long, value_enum)]
     pub config_location: Option<ConfigLocation>,
+
+    /// If set, this command will treat url as url to git repository
+    #[arg(long, default_value_t = false)]
+    #[builder(default)]
+    pub git: bool,
```

Comment:
Wondering if it would make sense to have two arguments `--git` and `--url` with type String instead of a positional argument and a boolean. Because at the moment the boolean does not require that the positional argument exists, and it would improve consistency overall